draft-ietf-tls-tls13.md

```diff
@@ -1817,14 +1817,17 @@ random
   See {{implementation-notes}} for additional information.
 
 legacy_session_id
-: Versions of TLS before TLS 1.3 supported a "session resumption"
+: {#legacy_session_id} Versions
```

Comment:
I don't think that you can use inline anchors like that, the parser does not seem to provide such a feature: https://github.com/cabo/kramdown-rfc2629